.tekton/odh-base-image-cuda-py311-c9s-push.yaml

```diff
@@ -25,14 +25,18 @@ spec:
     value: quay.io/opendatahub/odh-base-image-cuda-py311-c9s:{{revision}}
   - name: dockerfile
     value: base-images/cuda/12.6/c9s-python-3.11/Dockerfile.cuda
+  - name: build-platforms
+    value:
+    - linux/x86_64
+    - linux/arm64
   - name: path-context
     value: .
   - name: additional-tags
     value:
     - '{{target_branch}}-{{revision}}'
     - v12.6
   pipelineRef:
-    name: singlearch-push-pipeline
+    name: multiarch-pull-request-pipeline
```

Comment:
we should not use `pull-request-pipeline` in a `-push.yaml` tekton, no?

maybe copypaste

* https://github.com/opendatahub-io/notebooks/blob/main/.tekton/singlearch-push-pipeline.yaml

call it multiarch-push-pipeline.yaml, edit everything that needs to be edited to make it multiarch and use that?